src/pages/Setting/component/AllowanceModal.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// import { useNavigate } from 'react-router-dom';
+import SelectDropdown from './SelectDropdown';
+import CompleteBtn from './completeBtn';
+import './AllowanceModal.scss';
+
+const AllowanceModal = ({ isOpen, onClose }) => {
+  return (
+    <div className={`modal ${isOpen ? 'open' : ''}`}>
```

Comment:
isOpen이 false일 때 'close' 등의 className을 주면서 좀 더 명시적인 코드 작성을 해주는게 더 좋습니다. 혹은
```suggestion
    <div className={`${isOpen ? 'modal open' : 'modal'}`}>
```
예시처럼 처리할 수도 있겠네요